### PR TITLE
Add documentation for IPP printer component

### PR DESCRIPTION
**Pull request by Matte23**
**Description:** Documentation for the new component IPP Printer

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#23515

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/ipp_printer.markdown [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+layout: page
+title: "IPP Printer Sensor"
+description: "Instructions on how to integrate IPP printers into Home Assistant."
+date: 2019-04-28 12:34
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: System Monitor
```
Comment (klaasnicolaas) [inline source/_components/ipp_printer.markdown]:
```suggestion
ha_category:
  - System Monitor
  - Sensor
```

File: source/_components/ipp_printer.markdown [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+ha_release: 0.93
+ha_iot_class: Local Polling
+redirect_from:
```
Comment (klaasnicolaas) [inline source/_components/ipp_printer.markdown]: This is only for existing pages

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Comment by klaasnicolaas** [comment]
Pherhaps you could use this logo for the component? https://www.pwg.org/ipp/everywhere.html

**Comment by Matte23** [comment]
> Pherhaps you could use this logo for the component? https://www.pwg.org/ipp/everywhere.html

It's a good idea, but I'm not sure if non PWG members are allowed to use this logo.
I only found this document regarding the trademark, but it is about PWG members.
https://ftp.pwg.org/pub/pwg/general/process/wd-ippevelogo-policy-20151005.pdf

Maybe, if this icon is not suitable because of legal issues, I can use a generic internet printer logo or a simple image with "IPP Printer" written as a text

**Pull request closed without merging**